configuration/ldap_config.py

```diff
@@ -15,6 +15,9 @@
 AUTH_LDAP_BIND_DN = os.environ.get('AUTH_LDAP_BIND_DN', '')
 AUTH_LDAP_BIND_PASSWORD = os.environ.get('AUTH_LDAP_BIND_PASSWORD', '')
 
+# Set AUTH_LDAP_USER_DN_TEMPLATE
```

Comment:
I'd use something like `Set a string template that describes any user’s distinguished name based on the username.` here.